include/cuda/std/detail/libcxx/include/module.modulemap

```diff
@@ -356,6 +356,10 @@ module std [system] {
     header "memory"
     export *
   }
+  module memory_resource {
```

Comment:
I am unsure about this one here. I think that is only necessary for core headers and not our extensions.

Reply:
Removed